doc/concepts/morphology.rst

```diff
@@ -3,56 +3,117 @@
-linearly, resulting in either a cylinder (equal radii) or truncated cone (differing radii),
-centred at the line through the pair of points.
+  segment
+    A segment is a frustum (cylinder or truncated cone), with the center and radius at each
+    end defined by a pair of :term:`points <mpoint>`. In other words, in Arbor the radius between two points is interpolated
+    linearly, resulting in either a cylinder (equal radii) or truncated cone (differing radii),
+    centred at the line through the pair of points.
 
-.. csv-table::
+.. csv-table:: The properties of a :term:`segment`.
    :widths: 10, 10, 30
+   :align: center
 
-   **Field**,   **Type**, **Description**
+   **Field**,      **Type**,                           **Description**
    ``prox``,       :py:class:`point <arbor.mpoint>`,   the center and radius of the proximal end.
    ``dist``,       :py:class:`point <arbor.mpoint>`,   the center and radius of the distal end.
-   ``tag``,        integer, tag meta-data.
+   ``tag``,        integer (:term:`tag`),              "differentiate soma, axon, dendrite, etc."
+
+.. figure:: ../gen-images/term_segments.svg
+  :width: 300
+  :align: center
+
+  Four segments arranged head to toe.
+
+.. glossary::
+
+  branch
+    A branch is an unbranched sequence of :term:`segments <segment>`.
+
+.. figure:: ../gen-images/term_branch.svg
+  :width: 300
+  :align: center
+
+  A branch corresponding to the previous segments.
+
+.. glossary::
+
+  mlocation
+    A location is not a point in 3D space, but a point in the cable cell morphology's
+    coordinate system. It is defined by a specific branch, and a fraction that
 
-.. _morph-tag-definition:
+.. csv-table:: The properties of :term:`mlocation`.
+   :widths: 10, 10, 30
+   :align: center
+
+   **Field**,      **Type**,        **Description**
+   ``branch``,     integer,         id of the branch
+   ``pos``,        real (0≤pos≤1),  position along the length of the branch
+
+.. glossary::
+
+  cable
+    A cable is a subset of a :term:`branch`, and is thus defined as between two :term:`locations <mlocation>` on a particular branch.
 
-A *tag* is an integer label on every segment, which can be used to define disjoint
-regions on cells.
-The meaning of tag values are not fixed in Arbor, however we typically use tag values that correspond
-to SWC `structure identifiers <http://www.neuronland.org/NLMorphologyConverter/MorphologyFormats/SWC/Spec.html>`_.
+.. figure:: ../gen-images/term_cable.svg
+  :width: 300
+  :align: center
+
+  A cable shown in black is defined as a section :term:`located <mlocation>` between 0.2 and 0.8 of the length of the previous branch.
+
+.. note::
+
+  NEURON uses different nomenclature for segments and branches. What's called a *section* in NEURON corresponds to an Arbor :term:`segment`, and sometimes an Arbor :term:`branch`. The number of *segments* (`nseg`) that NEURON uses for control over discretisation corresponds to an Arbor :term:`control volume`.
```

Comment:
This is not actually true. I described earlier what Neuron segments and sections refer to.
Neuron section ~= Arbor branch 
Neuron segment ~= Arbor CV

Reply:
Thanks, I've updated this.